doubly_linkedlist/DoublyLinkedListOperations.java

```diff
@@ -24,6 +24,7 @@ void insertAtPostion(T data,int position) {
 			
 			if(position==0) {
 				newNode.next=head;
+				newNode.prev=null;
 				head.prev=newNode;
```

Comment:
Note: By default the next and prev pointers point to null when a new node is create. Its an extra line of code.